www/vercel.json

```diff
@@ -16,8 +131,118 @@
       "permanent": true
     },
     {
-      "source": "/docs/invalidateQuery",
-      "destination": "/docs/invalidateQueries",
+      "source": "/docs/adapters",
+      "destination": "/docs/server/adapters",
+      "permanent": true
+    },
+    {
+      "source": "/docs/api-gateway",
+      "destination": "/docs/server/adapters/aws-lambda",
+      "permanent": true
+    },
+    {
+      "source": "/docs/aws-lambda",
+      "destination": "/docs/server/adapters/aws-lambda",
+      "permanent": true
+    },
+    {
+      "source": "/docs/express",
+      "destination": "/docs/server/adapters/express",
+      "permanent": true
+    },
+    {
+      "source": "/docs/fastify",
+      "destination": "/docs/server/adapters/fastify",
+      "permanent": true
+    },
+    {
+      "source": "/docs/fetch",
+      "destination": "/docs/server/adapters/fetch",
+      "permanent": true
+    },
+    {
+      "source": "/docs/standalone",
+      "destination": "/docs/server/adapters/standalone",
+      "permanent": true
+    },
+    {
+      "source": "/docs/api-handler",
+      "destination": "/docs/server/api-handler",
+      "permanent": true
+    },
+    {
+      "source": "/docs/authorization",
+      "destination": "/docs/server/authorization",
+      "permanent": true
+    },
+    {
+      "source": "/docs/caching",
+      "destination": "/docs/server/caching",
+      "permanent": true
+    },
+    {
+      "source": "/docs/context",
+      "destination": "/docs/server/context",
+      "permanent": true
+    },
+    {
+      "source": "/docs/data-transformers",
+      "destination": "/docs/server/data-transformers",
+      "permanent": true
+    },
+    {
+      "source": "/docs/error-formatting",
+      "destination": "/docs/server/error-formatting",
+      "permanent": true
+    },
+    {
+      "source": "/docs/error-handling",
+      "destination": "/docs/server/error-handling",
+      "permanent": true
+    },
+    {
+      "source": "/docs/infer-types",
+      "destination": "/docs/server/infer-types",
+      "permanent": true
+    },
+    {
+      "source": "/docs/server",
+      "destination": "/docs/server/introduction",
```

Comment:
```suggestion
      "destination": "/docs/server/setup",
```

Reply:
Ah this one is worth discussion, there's context.

`/introduction` as a proper page is coming, and currently points at the generated index. Rather than add another redirect in a week it made sense to prepare for that new page and mean-time point at the generated page.

I'm going to jump ahead and revert the change, but definitely let me know your thoughts

Reply:
Ah okay. Yea lets do that then!